README.md

```diff
@@ -38,7 +38,7 @@ For a curated list of learning resources please check out [desireevl's repo](htt
 
 **C++**
 - [avaloni](https://github.com/avalon-lang/avaloni)  Programming language (interpreter) for classical-quantum hybrid computers.
-- [CUDA Quantum] (https://github.com/NVIDIA/cuda-quantum) - A quantum programming model for heterogeneous quantum-classical workflows.
+- [CUDA-Q] (https://github.com/NVIDIA/cuda-quantum) - Platform for accelerated quantum - classical applications on GPUs, CPUs and QPUs.
```

Comment:
I'm not sure why you added a space between "quantum" and "classical" in this line (and in line 50 below), but "quantum-classical" should be a single hyphenated term.